src/main/java/com/iota/iri/network/pipeline/TransactionProcessingPipelineImpl.java

```diff
@@ -69,6 +79,16 @@
     private BlockingQueue<ProcessingContext> broadcastStageQueue = new ArrayBlockingQueue<>(100);
     private BlockingQueue<ProcessingContext> replyStageQueue = new ArrayBlockingQueue<>(100);
 
+    private int getNumberOfThreads() {
+        int threads = 0;
+        for(Stage stage: Stage.values()) {
+            if(!IGNORED_STAGES.contains(stage)){
+                threads += 1;
+            }
+        }
+        return threads;
+    }
+
```

Comment:
You could have done it in a one liner:
` return Stages.values().length - IGNORED_STAGES.size()`

The above should also run faster (not that it matters much here)

Reply:
Btw, the reason I feel this is important is because any extra line of code is one more line for the maintainer to read

Since code is written once but read many times, these little changes throughout the codebase may save lots of time later on.